maestro-ios-driver/src/main/kotlin/xcuitest/api/OkHttpClientInstance.kt

```diff
@@ -0,0 +1,16 @@
+package xcuitest.api
+
+import okhttp3.ConnectionPool
+import okhttp3.OkHttpClient
+import java.util.concurrent.TimeUnit
+
+object OkHttpClientInstance {
```

Comment:
this will lead to multiple places using the same configuration for http connection, which is bad (some situations might require larger timeouts, some might require smaller pools, etc. I dont think it's a good idea

Reply:
Going to use the HttpClientBuilder you created on the other PR. Removing this